index.bs

```diff
@@ -668,10 +677,12 @@ Registry instances are initially created with the internal slots described in th
 1. Let _hook_ be GetMethod(_loader_, @@translate).
 1. Let _p_ be the result of transforming RequestFetch(_loader_, _key_) with a fulfillment handler that, when called with argument _payload_, runs the following steps:
   1. // TODO: metadata
-  1. Let _p1_ be the result of promise-calling _hook_(_key_, _payload_).
-  1. Return the result of transforming _p1_ with a fulfillment handler that, when called with argument _source_, runs the following steps:
+  1. Let _p0_ be the result of promise-calling _hook_(_key_, _payload_).
+  1. Let _p1_ be the result of transforming _p0_ with a new pass-through promise.
```

Comment:
`p1` is now the wrapping promise that can be resolved via `p0` or via `.provide()`, no matter which way it gets resolved, we will always update the state.